app/views/steps/respondent/under_age/edit.html.erb

```diff
@@ -1,12 +1,12 @@
 <% title t('.page_title') %>
```

Comment:
Hey, I totally forgot we still had this page for respondent. We have to remove it. Along with any logic left with regards to respondents. It is only the applicants that needs to run this logic. If the respondent is under age we now let them pass, there is no kick out like this.

Reply:
I will address that on a different PR

Reply:
Added a story for it https://mojdigital.teamwork.com/#/tasks/20340358